Resources/Prototypes/Alerts/alerts.yml

```
@@ -407,19 +407,43 @@
+    state: grabbed-choke
+  onClick: !type:StopBeingPulled { }
   name: alerts-pulled-name
   description: alerts-pulled-desc
+  minSeverity: 0
+  maxSeverity: 3
 
 - type: alert
   id: Pulling
-  icons: [ /Textures/Interface/Alerts/Pull/pulling.png ]
+  icons:
+  - sprite: /Textures/_White/Interface/Alerts/pull.rsi
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)